src/app/components/shared/baw-client/baw-client.component.ts

```diff
@@ -39,11 +45,28 @@ export class BawClientComponent extends withUnsubscribe() implements OnInit {
    * beginning with a `/`
    */
   @Input() public page: string;
+
   public url: SafeResourceUrl;
   public loading = true;
-  /** Used to track the height of the iframe */
-  private previousHeight = 0;
-  private iframeUpdateIntervalMs = 250;
+
+  @HostListener("window:message", ["$event"])
+  private onBawClientMessage(event: MessageEvent): void {
+    // Ignore events which are not from the old client
+    if (event.origin !== this.config.endpoints.oldClientOrigin) {
+      return;
+    }
+
+    try {
+      const meta: { height: number } = JSON.parse(event.data);
+
+      if (meta.height > 0) {
+        this.loading = false;
+        this.updateIframeSize(meta.height);
+      }
+    } catch (e: any) {
+      // Ignore message
```

Comment:
should log - never suppress totally

Reply:
The Baw client posts other messages which may not relate to this use case. For example, webpack posts messages about its state for some reason. I could log, but most of it should theoretically be useless